.github/workflows/kitchen.yml

```diff
@@ -211,14 +160,6 @@ jobs:
       - name: Run Test Kitchen
         working-directory: kitchen-tests
         run:  |
-          ruby -v
-          echo "Which ruby are we using?"
-          which ruby
-          cd /home/runner/work/chef/chef
-          bundle install
-          gem install kitchen
```

Comment:
We don't need to `cd` into the directory, the yml is already setting `working-directory` for this step which eventually does the same.